context/app/static/js/components/detailPage/MetadataTable/MetadataTable.tsx

```diff
@@ -32,7 +34,13 @@ function MetadataTable({ tableRows = [] as MetadataTableRow[], columns = default
             {tableRows.map((row) => (
               <TableRow key={row.key}>
                 <IconTooltipCell tooltipTitle={row?.description}>{row.key}</IconTooltipCell>
-                <TableCell>{row.value}</TableCell>
+                <TableCell>
+                  <Stack direction="row">
+                    {row.value}
+                    {row.key === 'age_value' ||
+                      (row.key === 'donor.age_value' && <DonorAgeTooltip donorAge={row.value} />)}
```

Comment:
These were the two matching values for this row I was able to find - not sure if we have all of the expected keys for the metadata table somewhere?

Reply:
It seems like this logic is nested in one of the utils and doesn't reflect the es doc. You could use `endsWith`?

Reply:
Good idea!